### DISCUSSION TITLE
500 infinite loop

### DISCUSSION THREAD

**Discussion by pk936** [vercel/next.js#37355] [Help]
On client side error/exception, the next JS shows 500 and keeps refreshing the 500 page for infinite time.
Is this a bug or is there a way to handle it ?

**Comment by icyJoseph** [comment]
Do you have any code to look at? This doesn't happen on my testing bench. My guess is that thew client side error is also triggering on 500? Would need to look at code to make an assessment.

**Comment by Evavic44** [comment]
Call me crazy but this loop also happens on a 404 error of a fresh `npx create-next-app` scaffold. Even without trying to handle errors manually using `not-found.tsx`.

Here's a [repository](https://github.com/Evavic44/infinite-error) to reproduce the code. I just created it now and tested the error.

> **Note**
> Error is only logged to the browser console and not on the local terminal.
